### ISSUE TITLE
Separate NMEA TCP configuration from NTRIP client 

### ISSUE THREAD

**Issue by carhargps** [sparkfun/SparkFun_RTK_Firmware#358]
At the moment if you want to configure NMEA TCP server/client you need to configure SSID and password of the Wifi you are using from the Rover NTRIP client tab. If you want to receive corrections using NTRIP client and broadcast them over TCP I think it's okay. In my case I use another RTK facet acting as a Base sending corrections using the ESP Now radio, so when I want to broadcast position data over TCP the rover is always trying to conect to a correction source. I think it would be better if you could do this things separately.

Also I think you can't configure it TCP server/client using wifi AP config.

**Comment by pyrog** [comment]
I think that Wifi settings are confusing (settings for rover, for base, for L band keys, for AP).

I suggest to have one tab/menu "network settings".
One could choose AP or one or more Wifi SSID.

In the second case, the device could connect automatically to the first SSID if available, and if not, the second… and so on.

Example:
```
Wifi_SSID_1 = MyPhone
Wifi_PW_1 = secret1

Wifi_SSID_2 = MyCompany
Wifi_PW_2 = secret2

Wifi_SSID_3 = MyHome
Wifi_PW_3 = secret3
```

**Comment by carhargps** [comment]
There is another problem. If the device doesn't have valid credentials set in the NTRIP Client, you are unable to connect to the device. I've tried to conect via TCP using matlab to view the raw messages and using valid credentials it connects almost instantly and if you don't set the correctly (because I don't want them, I have anothe source of corrections) it's imposible to establish the connection.

**Comment by nseidle** [comment]
> I suggest to have one tab/menu "network settings".

I am in agreeance. However

> One could choose AP or one or more Wifi SSID.

I don't quite understand. Is this what you mean?

Network Settings:
* SSID/PW 1 (entry boxes for SSID and PW like we have now)
* SSID/PW 2
* SSID/PW 3
* SSID/PW 4 (four seems like enough, no?)
* Device Config over AP or WiFi (Radio button between AP[default] or WiFi)

The RTK device would then start down the list trying each network.

If the user selects 'Config over WiFi', and then the user select the 'Config' menu from the device, the device would connect to the first SSID available and then the user could connect over any device on the same network (this is how I do dev work), rather than directly via the Access Point network.

**Comment by nseidle** [comment]
This is done:

![image](https://user-images.githubusercontent.com/117102/210083665-4a07bcb0-b567-45ba-bbc3-19773399895c.png)

Please give the [latest RC](https://github.com/sparkfun/SparkFun_RTK_Firmware/releases/tag/v2.7RC) a try (Dec 30th or later).

**Comment by nseidle** [comment]
TCP has now been fully extricated from NTRIP. 

![image](https://user-images.githubusercontent.com/117102/213528020-6ab31c90-ddfd-4717-a405-64ddd52072e3.png)

**Comment by nseidle** [comment]
This has been implemented in [v3.0 firmware](https://github.com/sparkfun/SparkFun_RTK_Firmware/releases/tag/v3.0).
